.bazelrc

```diff
@@ -20,3 +20,9 @@ build --workspace_status_command=util/get_workspace_status.sh
 # Generate coverage in lcov format, which can be post-processed by lcov
 # into html-formatted reports.
 coverage --combined_report=lcov --instrument_test_targets --experimental_cc_coverage
+
+# Because verilator requires it, the unit tests don't get much worse with it
+# set and because the tags = ["cpu:6"] doesn't work yet to configure the
+# verilator tests we can run 1 test per 4 cores to avoid verilator performance
+# falloff and timeouts.
```

Comment:
Could you reword this comment a bit?

I'm not so sure that verilator requires this... my machine (P920) has 72 cores, but bazel will only run a max of 10 test jobs in parallel.  I think this is more to prevent test parallelization from overloading smaller machines.